README.md

```diff
@@ -50,6 +50,20 @@ manages the following kind of services:
   Alternatively, any other application can be specified as a source of
   replication.
 
+![Composing Microservices](assets/composing-microservices.png "Composing Microservices")
```

Comment:
I'm wondering if this is the right spot. I think this image and paragraph should be above the first new paragraph that is introduce by #205 (in the nutshell section). with an additional sentence above the image, it should provide a good reading flow.

Reply:
Or it should go above the explanation of instance and replica to tell someone what how a composition of services should look like.

[Scratch my comment above](https://github.com/aixigo/PREvant/pull/206/files#r1743320841). I like this variant better.

Reply:
Done